core/src/main/java/com/kin/ecosystem/core/network/model/IncomingTransferRequest.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Kin Ecosystem
+ * Apis for client to server interaction
+ *
+ * OpenAPI spec version: 1.0.0
+ * 
+ *
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen.git
+ * Do not edit the class manually.
+ */
+
+package com.kin.ecosystem.core.network.model;
+
+import java.util.Objects;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * IncomingTransferRequest
+ */
+
+public class IncomingTransferRequest {
```

Comment:
Also doesn't like the name as it's not the actual request.

Reply:
changed